webpack.config.js

```diff
@@ -37,6 +37,16 @@ module.exports = {
     : {},
   module: {
     rules: [
+      {
+        test: /\.m?js$/,
```

Comment:
We never use `.mjs` extensions so this regex probably can be simplified to
```suggestion
        test: /\.js$/,
```